ratis-grpc/src/main/java/org/apache/ratis/grpc/server/GrpcClientProtocolService.java

```diff
@@ -48,8 +48,8 @@
 import java.util.function.Consumer;
 import java.util.function.Supplier;
 
-class GrpcClientProtocolService extends RaftClientProtocolServiceImplBase {
-  private static final Logger LOG = LoggerFactory.getLogger(GrpcClientProtocolService.class);
+public class GrpcClientProtocolService extends RaftClientProtocolServiceImplBase {
+  public static final Logger LOG = LoggerFactory.getLogger(GrpcClientProtocolService.class);
```

Comment:
I also encountered compile error while trying recent Ratis changes (RATIS-1515, etc.) with Ozone.  But I think exposing `LOG` as public only for Ozone test is bad practice.  We may be able to improve `TestRatisPipelineLeader` by using custom retry policy.